### PR TITLE
Governance: proposal content structure

### PR DESCRIPTION
**Pull request by Fraccaman**

### CODE REVIEW

File: src/architecture/m1/governance.md [hunk 1]
```diff
@@ -27,6 +27,22 @@ Each proposal will be stored in a sub-key under the internal proposal address. T
...
+  Authors: <authors email addresses> 
+  Discussions-To: <email address / link>
+  Created: <date created on, in ISO 8601 (yyyy-mm-dd) format>
+  License: <abbreviation for approved license(s)>
+  Abstract: <text>
+  Motivation: <text>
+  Details: <text>
+  Requires: <AIP number(s)> - optional field
+  Replaces: <AIP number> - optional field
+  Superseded-By: <AIP number> - optional field
```
Comment (cwgoes) [inline src/architecture/m1/governance.md:43]: how would this be added? governance proposals are immutable once created

File: src/architecture/m1/governance.md [hunk 2]
```diff
@@ -27,6 +27,22 @@ Each proposal will be stored in a sub-key under the internal proposal address. T
 
 `Author` address field will be used to credit the locked funds if the proposal is approved.
 
+The `content` value should follow a standard format. We leverage something similar to what is described in the [BIP2](https://github.com/bitcoin/bips/blob/master/bip-0002.mediawiki#bip-format-and-structure) document:
```
Comment (cwgoes) [inline src/architecture/m1/governance.md:30]: I suggest that we use JSON instead, it will make it easier to display in wallets, explorers, etc.

**Review by cwgoes — APPROVED** [review]
Concept ACK, a few suggested changes

File: src/architecture/m1/governance.md (hunk 2 above)
Comment (Fraccaman) [reply src/architecture/m1/governance.md:30]: we can keep the same structure but json formatted

File: src/architecture/m1/governance.md (hunk 1 above)
Comment (Fraccaman) [reply src/architecture/m1/governance.md:43]: right, we can keep `requires` and `replaces` but we may have to drop `Superseded-By`

**Review by cwgoes — APPROVED** [review]

**Pull request merged**